conf/.htaccess.assets.expiry

```diff
@@ -0,0 +1,9 @@
+# BEGIN core-standards:assetsExpiry
+<IfModule mod_expires.c>
```

Comment:
1. Is mod_expires available and active on all of our servers?

1. What's the overall distribution / availability between mod_expires and mod_headers?  Would we achieve a better compatibility with mod_headers?

Reply:
Then I guess we can use both, as some sources recommend.